docassemble/ALAutomatedTestingTests/data/questions/al_testing_files_to_push.yml

```diff
@@ -3,7 +3,6 @@ comment: |
   Templates use Mako to fill in the needed custom details.
   Templates are used as strings with PyGithub.
 ---
-# TODO: Make an interview that will generate a .env for people to download or maybe just copy/paste as it's a hidden file
 template: installer.env_example
```

Comment:
Should the env file example also get server_url added to it, with an example?

Reply:
Good eye spotting the discrepancy, though the comment on the issue will show you why it's not a blocker here - #143. Definitely something to improve in the future.